### PR TITLE
fix(docs): const logging = require.. contains binary

### PR DESCRIPTION
**Pull request by kinwa91**
This is preventing docs from generating correctly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "check": "gts check",
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r src/v2 build/src/v2 && cp -r protos build && cp test/*.js build/test",
-    "fix": "gts fix && eslint --fix 'samples/*.js' 'samples/**/*.js'",
+    "fix": "gts fix && eslint --fix 'samples/*.js' 'samples/**/*.js' 'src/**/*.js'",
```
Comment (JustinBeckwith) [inline package.json]: Just make this `eslint --fix '**/*.js'`.  If it picks up something it shouldn't we should have that in the `.eslintignore`.

File: src/v2/.eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (JustinBeckwith) [inline src/v2/.eslintrc.yml]: why?

File: package.json (hunk 1 above)
Comment (JustinBeckwith) [reply package.json]: I am generally confused.  Why are we not formatting the generated code?

Comment (kinwa91) [reply package.json]: I'm trying to make it format generated code :)

File: src/v2/.eslintrc.yml (hunk 2 above)
Comment (kinwa91) [reply src/v2/.eslintrc.yml]: We move the JS generated code under `build/`, which changes where it requires `package.json`.

Without this, `eslint` complains it cannot require `package.json`

File: package.json (hunk 1 above)
Comment (JustinBeckwith) [reply package.json]: lol sorry, misread.  Regardless, let's try to capture all the js :)

File: src/v2/.eslintrc.yml (hunk 2 above)
Comment (JustinBeckwith) [reply src/v2/.eslintrc.yml]: oh yeah!

**Pull request merged**